src/components/chatInput/index.js

```diff
@@ -40,8 +40,14 @@ import type { Dispatch } from 'redux';
 
 const QuotedMessage = connect()(
   getMessageById(props => {
+    console.log('props :', props);
     if (props.data && props.data.message) {
-      return <QuotedMessageComponent message={props.data.message} />;
+      return (
+        <QuotedMessageComponent
```

Comment:
@brianlovin @mxstbr i am modifying `QuotedMessage` to selected text. is it good idea ?? should we create seperate component like `QuotedSnippet` ??